### DISCUSSION TITLE
ModuleNotFoundError: No module named 'torch_geometric.utils.to_dense_adj'

### DISCUSSION THREAD

**Discussion by AAymanAbdelaal** [pyg-team/pytorch_geometric#9061] [Q&A]
Hello. When trying to import any of the torch_geometric_temporal modules, this error appears 'ModuleNotFoundError: No module named 'torch_geometric.utils.to_dense_adj'. Although, 'to_dense_adj' module is already imported using 'from torch_geometric.utils import to_dense_adj'. Any ideas?

**Comment by rusty1s** [comment]
Please see https://github.com/pyg-team/pytorch_geometric/discussions/9023#discussioncomment-8813817
